spec/api_spec.rb

```diff
@@ -422,6 +422,30 @@
     Cloudinary::Api.resources_by_moderation(:manual, :approved, :max_results => 1000)
   end
 
+  describe 'usage' do
+    let(:yesterday) { Date.parse((Date.today - 1).strftime('%d-%m-%Y')) }
```

Comment:
Can you please explain how this code is different from
`let(:yesterday) { Date.today - 1 }`
?